willa/etl/doc_proc.py

```diff
@@ -108,5 +108,9 @@ def split_all_docs(docs: list, chunk_size: int = 1000, chunk_overlap: int = 200)
 
 def embed_docs(chunked_docs: list, vector_store: VectorStore) -> list:
     """Embed documents using Ollama embeddings and store them in a vector store."""
-    document_ids = vector_store.add_documents(documents=chunked_docs)
+    langfuse = get_langfuse_client()
+    with langfuse.start_as_current_observation(as_type='embedding', name='embed_docs',
+                                               input=chunked_docs) as span:
```

Comment:
can we add something like the following? i'd like us to be able to filter out this as a trace external to a Willa interaction:

```suggestion
    with langfuse.start_as_current_observation(as_type='embedding', name='embed_docs',
                                               input=chunked_docs, metadata={'etl': True}) as span:
```